doc/source/whatsnew/v0.24.0.txt

```diff
@@ -159,6 +161,24 @@ This is the same behavior as ``Series.values`` for categorical data. See
 :ref:`whatsnew_0240.api_breaking.interval_values` for more.
 
 
+.. _whatsnew_0240.enhancements.bubble_plots:
+
+Bubble plots
+^^^^^^^^^^^^
+
+Bubble plots can now be produced with the :func:`plot.scatter()` method of ``DataFrame`` objects by passing a column name as an argument ``s`` to provide the data for bubble sizes. Bubble sizes automatically adjust to the maximum of the data, and a legend for bubble sizes also is automatically produced. (:issue:`22441`)
```

Comment:
Fix the ref to plot.scatter.

Reply:
Perhaps, "and the legend reflects the marker sizes".

Reply:
OK